index.html

```diff
@@ -73,7 +73,7 @@
         <div class="row center">
             <h3 class="white-text"> Molecular Simulation Design Framework (MoSDeF)</h3>
             <p class="subtitle white-text">
-                MoSDeF presents a set of extensible Python tools designed to facilitate the initialization, 
+                MoSDeF presents a set of extensible Python tools designed to facilitate the initialization,
```

Comment:
Suggested change to overview

MoSDeF is a multi-university collaboration focused on the development of software that enables more reproducible scientific workflows. MoSDeF presents a set of extensible Python tools designed to facilitate the initialization, atom-typing, and screening of soft matter systems using molecular simulation.   

MoSDeF is built around several key pieces of software. The mBuild library is used to generate initial configurations and is designed to allow users to build complex systems from smaller, interchangeable components that can be connected together.  The Foyer library provides a force field and simulator agnostic method for defining parameter usage, relying on SMARTS to express the chemical context, providing both human and machine-readable documentation of parameter usage. The GMSO library serves as a general-purpose container for storing parameterized simulations in simulation engine agnostic fashion and routines to create semantically correct input files for various simulation engines. 

@rsdefever Should probably add a description of mosdef cassandra library.